src/clib/pio.h

```diff
@@ -595,7 +653,7 @@ extern "C" {
                             MPI_Comm io_comm, int *iosysidp);
     int PIOc_get_numiotasks(int iosysid, int *numiotasks);
     int PIOc_Init_Intracomm(MPI_Comm comp_comm, int num_iotasks, int stride, int base, int rearr,
-                            int *iosysidp);
+                            rearr_opt_t *rearr_opts, int *iosysidp);
```

Comment:
Yeah, I was thinking of adding a new function to set flow control parameters after init (after merging this change).
Ok, In that case I will add a new interface to set the flow control parameters (initialize it to defaults in pio_init())